plugins/hls-retrie-plugin/src/Ide/Plugin/Retrie.hs

```diff
@@ -212,20 +211,12 @@ runRetrieInlineThisCmd state token RunRetrieInlineThisParams{..} = do
     --   Run retrie to get a list of changes
     --   Select the change that inlines the identifier in the given position
     --   Apply the edit
-    ast <- runActionE "retrie" state $
```

Comment:
All of these bindings were unused. Not sure if they can be removed (if something is relying on the side effects of these?) or not, but the tests are passing so.. :man_shrugging:

Reply:
I don't think they are nessary.

Reply:
If the tests are passing, let's remove them for now and wait for bug reports :)